python/ray/autoscaler/autoscaler.py

```diff
@@ -174,20 +179,21 @@ def _update(self):
 
         if nodes_to_terminate:
             self.provider.terminate_nodes(nodes_to_terminate)
-            nodes = self.workers()
+            nodes = self.managed_workers()
```

Comment:
Why do we need to make this call again?

Reply:
Shouldn't this be `self.workers()?`

Reply:
this naming is getting very confusing if we change the semantics of `self.workers()`, but the the point is that we want to work in terms of managed workers only.